libraries/chem-meta/src/parsing-utils/chemical-table-parser-base.ts

```diff
@@ -19,26 +9,26 @@ type CoordinateArrays = {
   z: Float32Array,
 }
 
-/** Base singleton for Molfile or Mol2 parser/handler */
-export abstract class ChemicalTableParserBase implements ChemicalTableParser {
-  protected constructor(file: string) {
-    this.init(file);
+/** Base for Molfile or Mol2 parser/handler */
+export abstract class ChemicalTableParserBase {
+  constructor(fileContent: string) {
+    this.init(fileContent);
```

Comment:
What's the point of having this method instead of doing all initialization in the constructor?